script.js

```diff
@@ -0,0 +1,156 @@
+let todoArr = [];
+let doneArr = [];
+
+/*로컬 스토리지에서 가져오기 함수*/
+const getLocalStorage = () => {
+    const localTodo = window.localStorage.getItem('todo');
+    const localDone = window.localStorage.getItem('done');
+    console.log(localTodo, localDone);
+
+    //로컬스토리지에 데이터가 있을때에만 가져온다
+    if (localTodo) {
+        todoArr = JSON.parse(localTodo);
+        todoArr.map((v) => createTodoElement(v, 'todo'));
```

Comment:
iterator는 되도록이면 명확한 네이밍을 갖는게 좋습니다.
```suggestion
        todoArr.map((todoItem) => createTodoElement(todoItem, 'todo'));
```

Reply:
습관처럼 쓰는 v, i 등등 고쳐야겠군요,,!!